src/styles/theme/defaultTheme.ts

```diff
@@ -669,5 +669,26 @@ export const defaultTheme = createTheme({
         },
       },
     },
+    MuiModal: {
+      styleOverrides: {
+        root: ({ theme }) => ({
+          '> .modal-content': {
+            position: 'absolute',
```

Comment:
Probably would have saved a few lines, but it would still had to be customized to an extent. Wanted to have a custom solution, without dragging too much code from Dialog into the codebase.